modules/ppcp-axo/src/AxoModule.php

```diff
@@ -349,6 +349,36 @@ function () use ( $c ) {
 			}
 		);
 
+		// we need to remove Fastlane when we are in order pay page
+		add_filter(
+			'woocommerce_available_payment_gateways',
+			/**
+			 * Param types removed to avoid third-party issues.
+			 *
+			 * @psalm-suppress MissingClosureParamType
+			 */
+			static function ( $methods ) use ( $c ) : array {
+				if ( ! is_array( $methods ) ) {
+					return $methods;
+				}
+
+				if (! is_wc_endpoint_url( 'order-pay' )) {
+					return $methods;
+				}
+
+
+				foreach ( $methods as $key => $method ) {
+					if ( $method && $method->id === 'ppcp-axo-gateway' ) {
+						unset( $methods[ $key ] );
+						break;
+					}
+				}
+
+
+				return $methods;
+			}
+		);
```

Comment:
Logic is solid, 💚  I'm so happy there's a relatively easy fix to this issue! 

If I were to nit-pick I'd suggest we simplify this code a bit. Something like:

```suggestion
	// Remove Fastlane on the Pay for Order page.
	add_filter(
    		'woocommerce_available_payment_gateways',
			/**
			 * Param types removed to avoid third-party issues.
			 *
			 * @psalm-suppress MissingClosureParamType
			 */
    		static function ( $methods ) {
        		if ( ! is_array( $methods ) || ! is_wc_endpoint_url( 'order-pay' ) ) {
            		return $methods;
        	}
        
        	// Remove Fastlane if present.
        	unset( $methods[AxoGateway::ID] );
        
        	return $methods;
    	}
	);
```